crates/common/src/fs.rs

```diff
@@ -49,6 +50,17 @@ pub fn read_json_file<T: DeserializeOwned>(path: &Path) -> Result<T> {
     serde_json::from_str(&s).map_err(|source| FsPathError::ReadJson { source, path: path.into() })
 }
 
+/// Reads the gzip file and deserialize it into the provided type.
+pub fn read_json_gzip_file<T: DeserializeOwned>(path: &Path) -> Result<T> {
+    // read the file into a byte array first
+    // https://github.com/serde-rs/json/issues/160
```

Comment:
outdated comment

Reply:
I updated comment in https://github.com/foundry-rs/foundry/pull/10190/commits/d305fffa516d05a2f790d7a8272618d442678722 pls lmk if I got your comment right